content/applications/websites/website/publish.rst

```diff
@@ -9,3 +9,4 @@ Publish
 
    publish/translate
    publish/multi_website
+   publish/on-premise_geo-ip-installation
```

Comment:
I looked at the toctree history and it's never been listed in the website doc...

Reply:
I added it to the toctree because the doc is inside the publish folder and creates warning messages. But if it is not a problem, then I'll simply remove It :)